src/mame/drivers/gigatron.cpp

```diff
@@ -59,7 +66,7 @@ void gigatron_state::prog_map(address_map &map)
 
 void gigatron_state::data_map(address_map &map)
 {
-	map(0x0000, 0x7fff).ram();
+	map(0x3fff, 0x20000).ram();
```

Comment:
What is it you're trying to do here? This won't pass validation (0x20000 is well outside the bounds of what the CPU can address) and, if it did, it wouldn't execute properly because it removes low RAM the program needs.

Reply:
Because it has 128K of RAM according to some documents that I've read.

Reply:
Wait, no. I am delusional. You're right. It only has 32K RAM.

Reply:
Even if it had 128K RAM, this would not be the correct way to add it. You'd have to make the final address be 2^n-1 or wherever it logically ends, and if only 32K or 64K were actually addressable at a time (like on many 8-bit computers) it would have to be banked in a more complicated way.

Reply:
Yeah. I reverted the changes.